src/Views/LoginPage/LoginPage/LoginPage.js

```diff
@@ -3,27 +3,30 @@ import Form from "../components/Form/Form";
 import TabBar from "../components/TabBar/TabBar";
 import "./styles.css";
 import PropTypes from "prop-types";
+import { login, register } from "../../../actions";
+import { connect } from "react-redux";
 
-function LoginPage({ loginUser }) {
-   const [login, setLogin] = useState({ email: "", password: "" });
-   const [signUp, setSingUp] = useState({ email: "", password: "" });
+function LoginPage({ login, register }) {
+   const [loginValues, setLoginValues] = useState({ email: "", password: "" });
+   const [signUpValues, setSingUpValues] = useState({ email: "", password: "" });
    const [tabBarButtonId, setTabBarButtonId] = useState("login");
 
    const handleLogin = (email, password) => {
-      setLogin({ email, password });
-      loginUser(email, password);
+      setLoginValues({ email, password });
+      login(email, password);
    };
 
    const handleSignUp = (email, password) => {
-      setSingUp({ email, password });
+      setSingUpValues({ email, password });
```

Comment:
You don't need to set Values here. They going to be updated on handleChanges

Reply:
done